examples/main/src/layer-examples.js

```diff
@@ -218,10 +219,10 @@ const PointDensityGridLayerExample = {
   props: {
```

Comment:
We could consider adding a separate `propTypes` object here, instead of adding special cases to the type deducer. When mature this support could be moved into the base `Layer` so that we would support a static `propTypes` field, somewhat similar to React), and this component could just read that field. @Pessimistress 

```
const PointDensityGridLayerExample = {
   layer: PointDensityGridLayer,
   propTypes: {
      coverage: {type: 'range', min: 0, max: 1}
   },
    props: {
```